src/txmempool.cpp

```diff
@@ -465,11 +465,12 @@ CTxMemPool::CTxMemPool(CBlockPolicyEstimator* estimator, int check_ratio)
     _clear(); //lock free clear
 }
 
-void CTxMemPool::ConnectManagers(gsl::not_null<CDeterministicMNManager*> dmnman)
+void CTxMemPool::ConnectManagers(gsl::not_null<CDeterministicMNManager*> dmnman, gsl::not_null<llmq::CInstantSendManager*> isman)
 {
     // Do not allow double-initialization
-    assert(m_dmnman == nullptr);
+    assert(m_dmnman == nullptr && m_isman == nullptr);
```

Comment:
[can be addressed in follow-up PR]

2 asserts are better than 1 assert with 2 conditions.
In case if assert will happen in current implementation it is impossible to understand which exactly variable has been not nullptr

```cpp
assert(m_dmnman == nullptr);
assert(m_isman == nullptr);
```

Reply:
Done, resolved in latest push!